app/Models/Bot/Commands/Division.php

```diff
@@ -1,34 +1,31 @@
 <?php
 
-namespace App\Models\Slack\Commands;
+namespace App\Models\Bot\Commands;
 
-use App\Models\Slack\Base;
-use App\Models\Slack\Command;
+use App\Models\Bot\Base;
+use App\Models\Bot\Command;
 
 /**
  * Class Search.
  */
 class Division extends Base implements Command
 {
-    public function __construct($data)
+    public function __construct($request)
     {
-        parent::__construct($data);
+        parent::__construct($request);
 
-        $this->request = $data;
+        $validated = $this->request->validate([
```

Comment:
The validated variable will contain the validated data so you could use this instead of the original data. Main reason you'd want to do this is to automatically discard any data passed that wasn't validated. 

Could probably move this down to the handle method or assign it to a property

Reply:
Resolved in 897e3a3